.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: node_js
+
 node_js:
   - "0.11"
+  - "0.12"
+  - iojs
+
+sudo: false
```

Comment:
It allows Travis to use the container-based infrastructure for faster build times.